### ISSUE TITLE
Support for UITraitCollection?

### ISSUE THREAD

**Issue by elstntho** [robertwijas/UISS#54]
Hi, 
are there any plans to support UITraitCollection, i.e. appearanceForTraitCollection: and  appearanceForTraitCollection:whenContainedInInstancesOfClasses: ?

Best regards,
Thomas

**Comment by robertwijas** [comment]
Yes. It's actually already implemented in 2.0, but I'm having a hard time finding motivation to finalize and publish it. But please keep nagging me about it, it should help :)

**Comment by elstntho** [comment]
Maybe I can support you in doing so - In my fork, I've made UISS compatible to iOS10 (i.e. replaced a couple of deprecated enums and method calls) and transformed the test cases to XCTest. If you don't mind, give me whatever you have for 2.0 and I'll try to make it ready for release.

Best regards,
Thomas
